Sync/Synchronizers/TabsSynchronizer.swift

```diff
@@ -176,15 +175,15 @@ open class TabsSynchronizer: TimestampedSingleCollectionSynchronizer, Synchroniz
 }
 
 extension RemoteTab {
-    public func toJSON() -> JSON? {
+    public func toDictionary() -> Dictionary<String, Any>? {
         let tabHistory = optFilter(history.map { $0.absoluteString })
```

Comment:
Might as well change this to `history.flatMap { $0.absoluteString }` while you're here.